stdlib/REPL/src/docview.jl

```diff
@@ -402,8 +402,9 @@ function symbol_latex(s::String)
 
     return get(symbols_latex, s, "")
 end
-function repl_latex(io::IO, s::String)
-    latex = symbol_latex(s)
+function repl_latex(io::IO, s0::String)
```

Comment:
```suggestion
function repl_latex(io::IO, s0::String)
    # This has rampant `Core.Box` problems (#15276). Use the tricks of
    # https://docs.julialang.org/en/v1/manual/performance-tips/#man-performance-captured
    # We're changing some of the values so the `let` trick isn't applicable. 
```

Reply:
Is the problem here linked to the fact that a `String` is not a real immutable object under the hood?

Reply:
>  better?

Yes that's great, thanks!

Reply:
No, it's just that it's captured by the closure *and* set in the main body.